travis/travis_install_nightly

```diff
@@ -113,13 +113,22 @@ pip install -q QUnitSuite codecov==2.0.15 coveralls==1.8.2
 # Use reference .coveragerc
 cp ${HOME}/maintainer-quality-tools/cfg/.coveragerc .
 
-echo "Getting addons dependencies"
-clone_oca_dependencies
-clone_result=$?
-if [ "$clone_result" != "0"  ]; then
-    echo "Error cloning dependencies"
-    exit $clone_result
-fi;
+MQT_DEP=${MQT_DEP:-OCA}
+if [[ "${MQT_DEP}" == "OCA" ]] ; then
+    echo "Getting addons dependencies"
+    clone_oca_dependencies
```

Comment:
I removed the exit code test because this script runs with `set -e` so it is dead code.